libraries/chain/nft_evaluator.cpp

```diff
@@ -24,6 +24,26 @@ void_result nft_metadata_create_evaluator::do_evaluate( const nft_metadata_creat
       const auto& ar_obj = (*op.account_role)(db());
       FC_ASSERT(ar_obj.owner == op.owner, "Only the Account Role created by the owner can be attached");
    }
+
+   // Lottery Related
+   if (!op.lottery_options) {
+      return void_result();
+   }
+   FC_ASSERT((*op.lottery_options).end_date > now || (*op.lottery_options).end_date == time_point_sec());
+   if (op.max_supply) {
+      FC_ASSERT(*op.max_supply >= 5);
+   }
+
+   for(auto lottery_id: (*op.lottery_options).progressive_jackpots) {
+      const auto& lottery_obj = lottery_id(db());
+      FC_ASSERT(lottery_obj.owner == op.owner, "Only the Owner can attach progressive jackpots");
+      FC_ASSERT(lottery_obj.is_lottery(), "Only lottery objects can be attached as progressive jackpots");
+      FC_ASSERT(lottery_obj.lottery_data->lottery_options.is_active == false, "Lottery should not be active");
```

Comment:
We need to remove this check because for NH5050, there is only one progressive lottery which starts in April and ends in November and there are multiple normal draws happening each month from April to October. A user can only purchase a ticket for a normal 5050 draw. When a user purchases a ticket for the normal draw, he gets an entry for the normal draw and an entry for the progressive draw. So, the progressive draw has to active in this case.